flutter/lib/src/integrations/native_app_start_integration.dart

```diff
@@ -62,8 +72,12 @@ class NativeAppStartIntegration extends Integration<SentryFlutterOptions> {
         if (options.automatedTestMode) {
           rethrow;
         }
+      } finally {
+        _frameCallbackHandler.removeTimingsCallback(timingsCallback);
```

Comment:
we only care about the first frame timing so after we processed it we can just remove the callback

Reply:
yes but let me add an additional widget test for this